src/config/model.c

```diff
@@ -1483,6 +1487,12 @@ static void clear_model(u8 full)
     }
     Model.ppmin_centerpw = 1500;
     Model.ppmin_deltapw = 400;
+#if HAS_EXTENDED_AUDIO
+    for (int i = 0; i < NUM_TIMERS; i++) {
```

Comment:
can this be replaced with `if (HAS_EXTENDED_AUDIO)`?  I'd like to avoid #if conditions where they aren't strictly needed